src/Compilers/Core/Portable/Diagnostic/Diagnostic.cs

```diff
@@ -559,6 +559,14 @@ internal virtual bool IsNotConfigurable()
         {
             return AnalyzerManager.HasNotConfigurableTag(this.CustomTags);
         }
+
+        /// <summary>
+        /// Returns true if this is an error diagnostic which cannot be suppressed and is guaranteed to break the build.
+        /// Only diagnostics which have default severity error and are tagged as NotConfigurable fall in this bucket.
+        /// This includes all compiler error diagnostics and specific analyzer error diagnostics that are marked as not configurable by the analyzer author.
+        /// </summary>
+        internal bool IsUnsuppressedError()
```

Comment:
>IsUnsuppressedError [](start = 22, length = 19)

IsUnsuppressableError ?

Reply:
Yes, that sounds like a better name. I will update.